CONTRIBUTING_DE.md

```diff
@@ -0,0 +1,180 @@
+    <strong>Beachte, du gibst nun einer Datei aus dem Internet die Berechtigung auf deinem Computer ausgeführt zu werden. Es ist Empfehlenswert in solchen Fällen sicherzustellen das du dem Code und Author vertraust. (Oder noch besser: verstehst, was der Code macht)</strong>
+
+```shell
+# Datei ausführbar machen
+chmod +x .git/hooks/pre-commit
+# Einen Verweiss auf die Datei erstellen, alternativ kannst du 'cp' anstelle von 'ln' verwenden um die Datei zu kopieren
+ln ./pre-commit.sh .git/hooks/pre-commit
+# Sieh dir an wie Lange die ausführung dauert und stelle sicher, dass alles funktioniert
+time .git/hooks/pre-commit
+```
+</details>
+
+#### Prüfliste :pencil:
+
+Wir schätzen jeden Beitrag zu unserem Projekt sehr und sind dankbar dafür! :heart:
+Um die Zusammenarbeit für alle reibungsloser und angenehmer zu gestalten,
+haben wir diese Checkliste zusammengestellt :scroll:.
+Durch das Prüfen dieser Punkte verbesserst du die Qualität und Konsistenz deiner Beiträge :sparkles: und beschleunigt den Überprüfungsprozess :rocket:.
+
+
+- [x] **Synchronisation mit dem zentralen Repository:** :arrows_counterclockwise: Stelle sicher, dass dein Fork auf dem neuesten Stand des zentralen Repositories ist, um eine reibungslose Zusammenführung zu ermöglichen. [GitHub-Dokumentation](https://docs.github.com/de/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork)
+- [x] **Lint:** :wrench: Stelle sicher, dass Linters auf alle neuen oder geänderten Dateien angewendet wurden.
+- [x] **Bedeutsame Änderungen:** :mag_right: Bestätige, dass jede geänderte Datei einen sinnvollen Inhalt beiträgt und vermeide unbedeutende Änderungen wie reine Anpassungen von Leerzeichen.
+- [x] **Testing:** :test_tube: Schreibe Tests für alle neuen Funktionen und aktualisiere bestehende Tests, wenn du Änderungen an Funktionalitäten vorgenommen hast.
+- [x] **Sprache & Rechtschreibung:** :book: Verwende Englisch für alle Variablennamen, Klassennamen, Funktionen, Kommentare usw., und stelle sicher, dass aller hinzugefügter Inhalt Rechtschreibprüfungen durchlaufen hat.
+- [x] **Sensible Informationen:** :no_entry: Überprüfe vor dem Einreichen noch einmal, um sicherzustellen, dass keine Passwörter, Zugangsdaten oder lokale Konfigurationen in deinen Änderungen vorhanden sind.
+- [x] **Kontinuierliche Integration:** :green_circle: Stelle sicher, dass der GitHub Actions CI-Build erfolgreich abgeschlossen werden kann, ohne Testfehler.
+
+## Datenbank :floppy_disk:
+
+### Verwende Dev-Data für die Lokale Entwicklung :construction_worker:
+Um die Entwicklung zu vereinfachen und Konsistenz in lokalen Umgebungen zu gewährleisten,
+bieten wir einen vordefinierten Datensatz an, der als 'Dev-Daten' bekannt ist.
+Dieser Datensatz ist darauf ausgerichtet, den Testprozess zu optimieren und eine diverse
+Datenabbildung zu erlangen um auch Randfälle zu Testen.
```

Comment:
Den zweiten Teil finde ich etwas holprig. Wie wärs mit:
> und damit wir beim Entwickeln immer eine breite Pallette an realistischen Daten sowie einige bekannte Edge Cases zu Verfügung haben.

Reply:
Ich habe angepasst auf:

> Dieser Datensatz ist darauf ausgerichtet, den Testprozess zu optimieren und damit wir beim Entwickeln immer eine breite Palette an realistischen Daten sowie einige bekannte Randfälle zur Verfügung haben.

Find ich immernoch etwas Holprig. Geht auch in die Thematik "Verdeutschung von Anglizismen"